src/sentry/static/sentry/app/components/events/realUserMonitoring.tsx

```diff
@@ -155,4 +157,12 @@ const Value = styled('span')<{failedThreshold: boolean}>`
   }};
 `;
 
+function prettyValue(record: Vital | undefined, value: number): string {
+  if (record && record.type === 'duration') {
+    return getDuration(value / 1000, 3);
+  }
+
+  return value.toFixed(3);
+}
```

Comment:
I think this was brought up before but something like this has been copied pasted in several places. Can they be consolidated somehow? Maybe not in this PR but in the future.

Reply:
We have a few similar type formatting functions in app/utils/discover/charts.tsx as well.

Reply:
I added `src/sentry/static/sentry/app/utils/measurements/index.tsx` to consolidate these functions that are specific to measurements/web vitals.